### ISSUE TITLE
Does no longer work for Thunderbird 60.x

### ISSUE THREAD

**Issue by ThomasHeinrichSchmidt** [alehro/message-notes-plus#1]
Very useful Add-on, sad it does not work any more. 
Can we do something to motivate you to update it for new Thunderbird?

**Comment by Sensor66** [comment]
Yes, very useful!  What would it take to get this updated?  In other words, where can I pay?

**Comment by ThomasHeinrichSchmidt** [comment]
I was able to move my Message Notes to XNote++, another note extension for Thunderbird that also works for TB 68 and higher.
You need to locate your messagenotesplus.sqlite in the TB profile folder to extract the notes. I can add technical details of the conversion if someone is interested.

**Comment by jamaisx** [comment]
absolutely interested, I tried to convert the script but I gave up !

**Comment by ThomasHeinrichSchmidt** [comment]
OK, I'll prepare a Readme file and a sample that you can use for your conversion, probably Thursday.

**Comment by ThomasHeinrichSchmidt** [comment]
I've released my converter under issue #3
